src/components/include/utils/macro.h

```diff
@@ -159,12 +159,15 @@
 
 #ifdef BUILD_TESTS
 #define FRIEND_TEST(test_case_name, test_name) \
-  \
-friend class test_case_name##_##test_name##_Test
+                                               \
+  friend class test_case_name##_##test_name##_Test
 #endif
 
 #if defined(QT_PORT)
-#define PLATFORM_INIT(argc, argv) QCoreApplication application(argc, argv)
+#define PLATFORM_INIT(argc, argv)                    \
+  QCoreApplication application(argc, argv);          \
+  QThreadPool* pool = QThreadPool::globalInstance(); \
+  pool->setMaxThreadCount(100)
```

Comment:
It will be performed within the [SDLWIN-308](https://adc.luxoft.com/jira/browse/SDLWIN-308)